### DISCUSSION TITLE
hide a button during checkout process

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3581] [Q&A]
Hello!

Is there an option to hide the button "Add tickets for a different date" in the checkout process ?
Please check the image below.
https://ibb.co/ftxrSRb

**Comment by wiffbi** [comment]
Not directly, but you could use the option „Directly redirect to check-out after a product has been added to the cart.“ found in event->Settings->General in the tab „Cart“ – that way, users will be directed to checkout immediately and the button for a different date is not shown.
